### ISSUE TITLE
Please rename "estimates" to "quotes"

### ISSUE THREAD

**Issue by Netzlichter** [crater-invoice/crater#727]
Please rename "estimates" to "quotes". I think estimates is not the right word and it is leading to translation issues for example in german :). @mohitpanjwani

**Comment by huseyinkogo** [comment]
Totally agree

**Comment by huseyinkogo** [comment]
Or there should a way to change to your need.

**Comment by Kwaku-Ananse** [comment]
yes I agree....or add quote since the 2 dont mean the same. A quote is more of an offer-price and estimate is just a rough idea, unconfirmed list of prices....therefore, you can add both options
